contracts/SportMarkets/Parlay/ParlayPolicy.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Inheritance
+import "../../interfaces/IParlayMarketsAMM.sol";
+import "../../interfaces/ISportPositionalMarket.sol";
+import "../../utils/proxy/solidity-0.8.0/ProxyOwned.sol";
+import "../../utils/proxy/solidity-0.8.0/ProxyPausable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+
+import "../../interfaces/ISportsAMM.sol";
+import "../../interfaces/IParlayMarketsAMM.sol";
+
+import "../../interfaces/IParlayPolicy.sol";
+
+import "../../interfaces/ITherundownConsumer.sol";
+import "../../interfaces/IGamesPlayerProps.sol";
+
+contract ParlayPolicy is Initializable, ProxyOwned, ProxyPausable {
+    IParlayMarketsAMM public parlayMarketsAMM;
+    ISportsAMM public sportsAMM;
+    address public consumer;
+    mapping(uint => uint) public restrictedMarketsCount;
+    // toBeRemoved:
```

Comment:
remove if to be removed

Reply:
indeed, to be removed prior to deployment on mainnet. 
Not removed due to proxy contract on OP goerli